config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -23,6 +23,25 @@ postsubmits:
         secret:
           secretName: oauth-token
   kubernetes/test-infra:
+  - name: post-test-infra-deploy-prow
```

Comment:
This should only run on the `master` branch.

Reply:
Done